charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/es-statefulset.yaml

```diff
@@ -0,0 +1,65 @@
+# Elasticsearch deployment itself
+apiVersion: {{ include "statefulsetversion" . }}
+kind: StatefulSet
+metadata:
+  name: elasticsearch-logging
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: elasticsearch-logging
+    role: logging
+spec:
+  serviceName: elasticsearch-logging
+  selector:
+    matchLabels:
+      app: elasticsearch-logging
+      role: logging
+  replicas: {{ .Values.elasticsearchReplicas }}
+  template:
+    metadata:
+      labels:
+        app: elasticsearch-logging
+        role: logging
+    spec:
+      containers:
+      - name: elasticsearch-logging
+        image: {{ index .Values.images "elasticsearch-oss" }}
+        imagePullPolicy: IfNotPresent
+        resources:
+          # need more cpu upon initialization, therefore burstable class
+          limits:
+            cpu: 1000m
+            memory: 1.5Gi
```

Comment:
This memory limit is too low for Elasticsearch, https://www.elastic.co/guide/en/elasticsearch/guide/master/hardware.html
This should be configurable to allow people to properly set the limits to suit their needs.
Also will be nice to be explicit with the current value of ES_HEAP_SIZE

Reply:
@epierotto thanks for the hint. We plan to start with static settings for now and are investing into vertical pod autoscaling for these components. We envision a fully automated scale mechanism of the control planes based on the load/needs.